backend/src/estark/mod.rs

```diff
@@ -65,18 +67,18 @@ fn create_stark_struct(degree: DegreeType, hash_type: &str) -> StarkStruct {
     }
 }
 
-type PatchedConstants<F> = Vec<(String, Vec<F>)>;
+type Constants<F> = Vec<(String, Vec<F>)>;
```

Comment:
Is this typedef needed?

Reply:
Yes, it was already there thanks to clippy